apps/web/src/pages/DirectivePage/DirectivePage.test.tsx

```diff
@@ -0,0 +1,19 @@
+/**
+ * @jest-environment jsdom
+ */
+import "@testing-library/jest-dom";
+import React from "react";
+import { renderWithProviders, axeTest } from "@gc-digital-talent/jest-helpers";
+
+import DirectivePage from "./DirectivePage";
+
+const renderDirectivePage = () => {
+  return renderWithProviders(<DirectivePage />);
+};
+
+describe("DirectivePage", () => {
```

Comment:
Could you add some dead-link tests for the external pages and forms to Cypress, too?

Reply:
Okay, tests were spread out across multiple commits while I fixed some stuff related to them 😛 

- [add e2e test for directive a11y + form downloads](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6143/commits/7eaabc58bdd099ac75551528680b1619c91945e7)
- [fix e2e verify downloads](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6143/commits/5b6c7def3301d62f2ddcbdca38b081787d973e1e)
- [remove duplicate verify download commands](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6143/commits/0ab168be27e1f4ef0e6b26f5039dd516e26a12d5)